sql/ddl/create/foreign-key.md

```diff
@@ -75,15 +76,78 @@ CREATE TABLE employees (
 );
```

Comment:
I think the line above will fail the check of code-length < 44 chars

Reply:
yeah, I've split the entire `CONSTRAINT` definition over three lines to by-pass this